docker-compose.yml

```
@@ -160,14 +160,19 @@ services:
       - ./etc/index-page:/var/www/index-page:ro
       # public IPL assets, e.g. traffic data (served on :80)
       - ./var/www/datasets:/var/www/datasets:ro
+      # password protected webcam data (served on :80)
```

Let's make it clear that the password protection happens in front of `caddy`.

```suggestion
      # webcam data (served on :80, password-protected by the `ingress` service)
```

---

Added